src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/ValueProviderResult.cs

```diff
@@ -3,227 +3,372 @@
 
         /// <summary>
-        /// Instantiates a new instance of the <see cref="ValueProviderResult"/> class with given
-        /// <paramref name="rawValue"/>. Initializes <see cref="Culture"/> to
-        /// <see cref="CultureInfo.InvariantCulture"/>.
+        /// A <see cref="ValueProviderResult"/> that represents a lack of data.
         /// </summary>
-        /// <param name="rawValue">The <see cref="RawValue"/> value of the new instance.</param>
-        public ValueProviderResult(object rawValue)
-            : this(rawValue, attemptedValue: null, culture: _staticCulture)
-        {
-        }
+        public static ValueProviderResult None = new ValueProviderResult(new string[0]);
 
         /// <summary>
-        /// Instantiates a new instance of the <see cref="ValueProviderResult"/> class with given
-        /// <paramref name="rawValue"/>, <paramref name="attemptedValue"/>, and <paramref name="culture"/>.
+        /// Creates a new <see cref="ValueProviderResult"/> using <see cref="CultureInfo.InvariantCulture"/>.
         /// </summary>
-        /// <param name="rawValue">The <see cref="RawValue"/> value of the new instance.</param>
-        /// <param name="attemptedValue">The <see cref="AttemptedValue"/> value of the new instance.</param>
-        /// <param name="culture">The <see cref="Culture"/> value of the new instance.</param>
-        public ValueProviderResult(object rawValue, string attemptedValue, CultureInfo culture)
+        /// <param name="value">The submitted value.</param>
+        public ValueProviderResult(string value)
         {
-            RawValue = rawValue;
-            AttemptedValue = attemptedValue;
-            Culture = culture ?? _staticCulture;
-        }
+            if (value == null)
+            {
+                Value = null;
+                Values = None.Values;
+            }
+            else
+            {
+                Value = value;
+                Values = null;
+            }
 
-        /// <summary>
-        /// <see cref="string"/> conversion of <see cref="RawValue"/>.
-        /// </summary>
-        /// <remarks>
-        /// Used in helpers that generate <c>&lt;textarea&gt;</c> elements as well as some error messages.
-        /// </remarks>
-        public string AttemptedValue { get; }
+            Culture = _invariantCulture;
+        }
 
         /// <summary>
-        /// <see cref="CultureInfo"/> to use in <see cref="ConvertTo(Type)"/> or
-        /// <see cref="ConvertTo(Type, CultureInfo)"/> if passed <see cref="CultureInfo"/> is <c>null</c>.
+        /// Creates a new <see cref="ValueProviderResult"/>..
         /// </summary>
-        public CultureInfo Culture { get; }
+        /// <param name="value">The submitted value.</param>
+        /// <param name="culture">The <see cref="CultureInfo"/> associated with this value.</param>
+        public ValueProviderResult(string value, [NotNull] CultureInfo culture)
```

Comment:
Why impose the new requirement on `culture`? Was previously fine to pass `null`.